posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Epiverse-TRACE tools for analysing ongoing Mpox outbreak"
+author:
+  - name: "James Azam"
+    orcid: "0000-0001-5782-7330"
+date: "2024-06-19"
```

Comment:
```suggestion
date: "2024-07-01"
```

We are trying to pace our blog posts a bit more, and already have one scheduled for June 24th. Suggested date as we've been adding them on Monday mostly for now (no reason why Monday, just to have something).

Reply:
Don't think we should worry too much about staggering posts (don't think we have an e-mail list we're sending to automatically, so no spam risk) – suggest we do whatever is lowest friction/admin and gets finished content out there soonest, given information has high time value during outbreaks.

Reply:
Yes, I agree this needs to be out asap else it'll become untimely.